queue_services/entity-emailer/requirements.txt

```diff
@@ -67,7 +67,7 @@ rsa==4.7.2
 semver==2.13.0
 sentry-sdk==1.20.0
 six==1.15.0
-SQLAlchemy==1.3.24
+SQLAlchemy==1.4.44
```

Comment:
update the SQLAlchemy version to make it compatible with the custom sql-versioning